src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  padding: 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: white;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.imac img {
+  margin-top: 16px;
+  width: 160px;
+  height: 134px;
+}
```

Comment:
Don't create another block. Just create element card__image and you can remove div that wraps the img tag